views/partials/header.ejs

```diff
@@ -20,31 +20,25 @@
         </li>
         <% if (user.role === 'candidate' && !user.candidateProfile) { %>
         <li class="nav-item">
-          <a class="nav-link" href="<%=`/user/${user._id}/create_candidate` %>"> New Candidate </a>
+          <a class="nav-link" href="<%=`/user/${user._id}/create_candidate` %>"> New candidate </a>
         </li>
         <% } else if (user.role === 'recruiter') { %>
-        <li class="nav-item">
-          <a class="nav-link" href="<%=`/user/${user._id}/create_offers` %>"> Add job </a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="<%=`/user/${user._id}/offers` %>">Your Jobs </a>
-        </li>
         <% } %>
```

Comment:
this else if condition is empty and can be deleted then :P

Reply:
i was trying to delete it but i got an error, dunno why so i left it there 🤪